pkg/utils/dummies.go

```diff
@@ -23,6 +23,7 @@ func NewDummyCommon() *common.Common {
 		Log:        NewDummyLog(),
 		Tr:         &tr,
 		UserConfig: config.GetDefaultConfig(),
+		AppState:   config.GetDefaultAppState(),
```

Comment:
I feel it's good enough to instantiate a default-constructed app state in those tests that need one.